active/dceu.md

```diff
@@ -2,11 +2,18 @@
 
 ## Scope of responsibilities
```

Comment:
I’m surprised there isn’t a line item about participating in the call for proposal and vetting process, beyond the group being a point of contact. Is that intentional?

Reply:
I'm not sure what you mean, there are two line items about exactly that:

> - Provide advice and assitance on creating and executing the Call for Proposal process for future conferences
> - Assist the board in vetting incoming proposals and make recommendations for how to approach the decision (the final decision is always made by a board vote)

I deliberately phrased them in a way that the group assists the board, not takes over responsibility for these steps, as my impression wasn't that the board wants to delegate these things entirely, just have a source for knowledgeable input.

Reply:
My bad, I just missed it :) This is exactly what I was expecting.